install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -31,9 +31,7 @@ global:
     # disable access log for sidecar.
     accessLogFile: "/dev/stdout"
 
-    # If set, newly injected sidecars will have core dumps enabled. Core dumps will always be written to the same
-    # file to prevent storage filling up indefinitely. Add a timestamp option to core_pattern to keep all cores:
-    # e.g. sysctl -w kernel.core_pattern=/var/lib/istio/core.%e.%p.%t
+    # If set, newly injected sidecars will have core dumps enabled.
```

Comment:
@ostromart I thought you said you would want to prevent users to tweak the core pattern in previous comments, that's why I removed those instructions. Want me to add them back?

Reply:
Yes please. I meant that we don't want to add a helm parameter, but if users want to tweak it we should have this comment for them to edit the template or yaml directly.